From 838e0f96f06fff524aa3a8f6621dcc5b4d8d77f3 Mon Sep 17 00:00:00 2001
From: Dylan Yip <dylan.yip@xilinx.com>
Date: Tue, 9 Mar 2021 12:36:45 -0800
Subject: [PATCH 1774/1852] drm: xlnx: mixer: Print out mixer CRTC id

commit 73f5a606fba08d3b1deb0a97f386f980e190c7c1 from
https://github.com/Xilinx/linux-xlnx.git

If you have multiple CRTC connected to single encoder, it becomes
confusing to know which CRTC is which. So print out mixer CRTC id on
register.

Signed-off-by: Dylan Yip <dylan.yip@xilinx.com>
Signed-off-by: Jianqiang Chen <jianqiang.chen@xilinx.com>
Acked-by: Varunkumar Allagadapa <varunkumar.allagadapa@xilinx.com>
State: pending
Signed-off-by: Yaliang Wang <Yaliang.Wang@windriver.com>
---
 drivers/gpu/drm/xlnx/xlnx_mixer.c | 2 ++
 1 file changed, 2 insertions(+)

diff --git a/drivers/gpu/drm/xlnx/xlnx_mixer.c b/drivers/gpu/drm/xlnx/xlnx_mixer.c
index b4471c1fa0b3..a0931be832b9 100644
--- a/drivers/gpu/drm/xlnx/xlnx_mixer.c
+++ b/drivers/gpu/drm/xlnx/xlnx_mixer.c
@@ -2879,6 +2879,8 @@ static int xlnx_mix_crtc_create(struct xlnx_mix *mixer)
 	crtc->get_cursor_height = &xlnx_mix_crtc_get_max_cursor_height;
 	crtc->get_cursor_width = &xlnx_mix_crtc_get_max_cursor_width;
 	xlnx_crtc_register(mixer->drm, crtc);
+	dev_info(mixer->drm->dev,
+		 "Registered mixer CRTC with id: %d\n", crtc->crtc.base.id);
 
 	return 0;
 
-- 
2.31.1

